test/compare-images.test.ts

```diff
@@ -1,12 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-const fs = require('fs');
-const { PNG } = require('pngjs');
-const useBrowser = require('../src/use-browser').default;
-const { ScreenshotPageObject } = require('../src/page-objects');
-const { cropAndCompare, parsePng } = require('../src/image-utils');
+import fs from 'fs';
+import { PNG } from 'pngjs';
+import useBrowser from '../src/use-browser';
+import { ScreenshotPageObject } from '../src/page-objects';
+import { cropAndCompare, parsePng } from '../src/image-utils';
+import './utils/setup-local-driver';
```

Comment:
I removed this import from `setupFilesAfterEnv` because it does not allow to mock `webdriverio` object.

Currently we explicitly import this setup file in every suite where we use the real webdriver